### PR TITLE
Multi Insert

### PR DESCRIPTION
**Pull request by taxio**
#364 

Hi, aarondl.
I want sqlboiler to support Multi Insert (e.g `INSERT INTO "hoge" VALUES (a, b, c), (d, e, f)`).

Multi insert is required even in other libraries. https://github.com/jmoiron/sqlx/issues/134 

It is my first time to send PR to OSS, so please tell me anything if there is a fix.

### CODE REVIEW

**Comment by aarondl** [comment]
So I just double-checked and ensured that all of our database engines support this type of insert (and it looks like they do). I'll probably accept this PR with some tweaks. But first a thorough review. I'll find time when I can to get that done. Thanks for submitting it.

**Comment by aarondl** [comment]
It's acceptance is also conditional on whether or not we can actually populate the default values correctly. I have a feeling for `mysql` this is impossible.

**Comment by taxio** [comment]
Thanks for comments.

I don't understand enough about mysql...
Since I am in your slack, please feel free to advise.

File: templates/15_insert.go.tpl [hunk 1]
```diff
@@ -201,3 +201,168 @@ CacheNoHooks:
...
+	{{- end}}
+	if err != nil {
+		return errors.Wrap(err, "{{.PkgName}}: unable to insert into {{.Table.Name}}")
+	}
+	{{if $canLastInsertID -}}
+	var lastID int64
+	{{- end}}
+
+	{{if $canLastInsertID -}}
+	lastID, err = result.LastInsertId()
```
Comment (aarondl) [inline templates/15_insert.go.tpl]: If this code is not looped over as the rest of it is, there's no chance that mysql values will be returned for all inserted rows.

**Comment by taxio** [comment]
I fixed getting default values.


Maybe, I mistook the operation of git, so if you need, I will reissue PR later...

**Comment by aarondl** [comment]
Hey @taxio, can you clean up this PR first? There's too much noise in the diff.

**Pull request closed without merging**